### ISSUE TITLE
Officially vet and add TinySlider

### ISSUE THREAD

**Issue by timwright12** [10up/wp-component-library#197]
https://github.com/ganlanyuan/tiny-slider

**Comment by timwright12** [comment]
Plugin should be validated to pass WCAG 2.1 AA, then added to the library as a recommendation

**Comment by timwright12** [comment]
Assigned to Treighton

**Comment by smy315** [comment]
Just curious and I don't have a strong feeling about this. Any reason for choosing TinSlider over Flickity? I've used both but have used Flickity more simply based on the fact that it has less issues and used/watched more. 

![Screen_Shot_2020-01-28_at_11_30_38_AM](https://user-images.githubusercontent.com/2092765/73285286-ca7f7400-41c3-11ea-82f4-da6ab077d362.png)

Again, I've used both and don't have a strong preference. Just asking based on GitHub stats. Sorry if there was a discussion for this already or if this was considered.

Never validated accessibility for both but I do know that Flickity does provide "accessibility" options (which I know has some issues).

**Comment by timwright12** [comment]
@smy315 might as well vet them both
